### PR TITLE
One liner for installing the application.

### PR DESCRIPTION
**Pull request by MichaelDimmitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-2. Clone locally using `git clone https://github.com/schvetsov/ArtistDashboard.git`
-3. Install dependencies using `npm install`
-4. Run tests using `npm test`
-5. Start the server `npm start`
-6. Navigate to app in [browser](http://localhost:3000)
+```
+git clone https://github.com/schvetsov/ArtistDashboard.git &&
+npm install &&
+npm test &&
+npm start
```
Comment (schvetsov) [inline README.md:9]: Looks good but missing 'cd ArtistDashboard'

**Review by schvetsov — CHANGES_REQUESTED** [review]

**Comment by schvetsov** [comment]
Thanks for submitting a change for my project it really means a lot

File: README.md (hunk 1 above)
Comment (MichaelDimmitt) [reply README.md:9]: Nice catch! implemented in most recent commit.

Comment (MichaelDimmitt) [reply README.md:9]: @schvetsov

**Comment by schvetsov** [comment]
I see that we switched from a sequence of commands to a one line implementation. 

I approve, verified commands work correctly.

**Review by schvetsov — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-3. Install dependencies using `npm install`
-4. Run tests using `npm test`
-5. Start the server `npm start`
-6. Navigate to app in [browser](http://localhost:3000)
+```
+git clone https://github.com/schvetsov/ArtistDashboard.git &&
+cd ArtistDashboard &&
+npm install &&
+npm test &&
+npm start
```
Comment (schvetsov) [inline README.md:9]: looks good

**Review by schvetsov — APPROVED** [review]

**Pull request merged**